### DISCUSSION TITLE
How do I use these channels with TiviMate?

### DISCUSSION THREAD

**Discussion by MintJamsLP** [iptv-org/community#147] [Q&A] [answered]
Hello,

I looked around and didn't see any guide to how to make this work on a player that is not a PC. I looked in the discussions tab and there is no FAQ, guide, or other writeup about this. I have iptv with Tivimate and have never had a problem adding a playlist, but these look like a different format and they don't seem to work when adding them.

How do I add a playlist? Is it Xstream code or m3u? What links am I adding? The main links (sorted by country/category/language) or the links further down? are they all the github links?

Thank you

**Comment by MintJamsLP** [comment]
I also couldn't get any of the links to work with VLC either. None of them would open.

**Comment by BellezaEmporium** [comment, marked as answer]
The links you're supposed to use for both TiVimate or VLC is the links that are in the README below. They're made to be used in any player. The playlists we have is for referencing links, as it reconstructs the playlists for the public afterwards.

**Reply by MintJamsLP** [reply]
thank you, I was not able to get anything to work within VLC, but I was able to get it working on TiViMate

**Reply by BellezaEmporium** [reply]
Perfect, Issue solved, locking this conversation.
